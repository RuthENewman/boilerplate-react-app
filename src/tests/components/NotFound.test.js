import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

test('renders Not Found page correctly', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
