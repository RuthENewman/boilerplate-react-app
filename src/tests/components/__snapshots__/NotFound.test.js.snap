// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Not Found page correctly 1`] = `
<div>
  404 page not found - try the 
  <Link
    to="/"
  >
    home page
  </Link>
</div>
`;
